src/polyfills.ts

```diff
@@ -19,19 +19,24 @@
  */
 
 /** IE9, IE10 and IE11 requires all of the following polyfills. **/
```

Comment:
if we do not support IE, why include these polyfills?

Reply:
We don't fully support it, but if we can make basic features work with these widely used and well supported workarounds, it seems a good solution to have an okay user experience for IE users with minimal effort from our side. It's kind of a tradeoff...

Reply:
just that we adding these imports, as well as the lib for detecting IE, further increases the overhead, plus all the material icons and fonts we included in the app, things add up.

Reply:
ideally we want sth. light when built for production, the lighter the better

Reply:
and btw, I hate IE, so should everyone else :D

Reply:
I agree with you totally on this one!